docs/application/native/guides/multimedia/overview.md

```diff
@@ -1,9 +1,9 @@
-# Media and Camera
+# Multimedia
 
 
-The media and camera features introduce how you can use multimedia features in your application.
+Multimedia features show how an application can handle everything related to multimedia resources, such as images, audio, video, camera, radio, and more.
```

Comment:
We have a writing style for guides.

How about the following sentence?
_Multimedia introduce how you can use control multiple forms of media such as multimedia resources, such as images, audio, video, camera, and radio_.

Reply:
regarding the sentence you wrote..
1. you can use control -> you can control
2. Two "such as" which is not good.

Do you mean the style is like "... introduce how you can ... " ?

Reply:
@sangchul1011 
I'm sorry. There is a mistake.
I mean
_Multimedia introduce how you can control multimedia resources such as, images, audio, video, camera, and radio._

If it is okay, please update your change.